mods/d2k/rules/structures.yaml

```diff
@@ -953,6 +983,8 @@ research_centre:
 		Footprint: _X_ xxx XXX ===
 		Dimensions: 3,4
 		LocalCenterOffset: 0,-512,0
+	Encyclopedia:
+		Description: The IX Research Center provides technology upgrades for structures and vehicles.  This facility is required for production of a number of advanced special weapons and prototypes.\n\nThe Ix Research Center is vulnerable to most types of weapons.
```

Comment:
There's a second `Ix Research Center` in this desc that needs the capitalisation fix.

Reply:
Done.